Commonalities/documentation/Glossary.md

```diff
@@ -1,11 +1,12 @@
+
 # Glossary
 
 | Term      | Description | Usage in API (Parameter/Field) | Alternative developer-friendly terms |
 | ------------ | ----------- | ----------- |  ----------- |
-| **Device** | End-user equipment able to connect to a mobile network. Examples of devices include smartphones or IoT sensors/actuators.   | device |  |  |
+| **Device** | End-user equipment able to connect to a mobile network. Examples of devices include smartphones or IoT sensors/actuators.   | device |  |  |
```

Comment:
Proposal to omit "mobile" from the definition. We have already interest to use e.g. QualityOnDemand also in fixed networks and also APIs which are addressed especially devices in fixed networks, like https://github.com/camaraproject/HomeDevicesQoD 

```suggestion
| **Device** | End-user equipment able to connect to a network. Examples of devices include smartphones, home gateways or IoT sensors/actuators.   | device |  |  |
```